backend/elite2Api.js

```diff
@@ -23,6 +23,15 @@ const searchOffenders = (req, res) => gateway.getRequest({
   // NB response.headers['total-records']
 });
 
```

Comment:
would it be clearer to call the other one "searchOffendersWithResultLimit"

Reply:
a bit - i'll change it